test/Bond.spec.ts

```diff
@@ -592,6 +621,17 @@ describe("Bond", () => {
                 collateralToReceive: config.collateralTokenAmount.sub(1),
               });
             });
+
+            it("fails to withdraw when called by non-owner", async () => {
```

Comment:
Can we remove the test for this in each possible state, only have it for the active state? I don't think this adds any additional security.

Reply:
fixed in https://github.com/porter-finance/v1-core/pull/303/commits/ee9ab0b8fdcdd351120329e0d926d42c61976c36